db/package.json

```diff
@@ -14,6 +14,7 @@
   "dependencies": {
     "pg": "^8.12.0",
     "postgrator": "^7.2.0",
+    "spark-api": "https://github.com/filecoin-station/spark-api/archive/3de5c3325ef6cc40df02769b96957d33279d38c1.tar.gz",
     "spark-evaluate": "filecoin-station/spark-evaluate#main"
```

Comment:
FYI: I cannot use the git/GitHub shorthand for `spark-api` because spark-api has workspaces, and npm refuses to install the entire project when running `npm ci` limited to a single spark-stats workspace. I think it's a bug in npm, but 🤷🏻 

```
$ NPM_CONFIG_WORKSPACE=stats npm ci
(...)

npm error code 1
npm error git dep preparation failed
npm error command /opt/hostedtoolcache/node/20.16.0/x64/bin/node /opt/hostedtoolcache/node/20.16.0/x64/lib/node_modules/npm/bin/npm-cli.js install --force --cache=/home/runner/.npm --prefer-offline=false --prefer-online=false --offline=false --no-progress --no-save --no-audit --include=dev --include=peer --include=optional --no-package-lock-only --no-dry-run
npm error npm warn using --force Recommended protections disabled.
npm error npm error No workspaces found:
npm error npm error   --workspace=stats
```